python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1692,8 +1692,12 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
         # (to make it simpler!)
         # Output units: electron (adu if doBootstrap=True)
         if self.config.doDeferredCharge:
-            self.log.info("Applying deferred charge/CTI correction.")
-            self.deferredChargeCorrection.run(ccdExposure, deferredChargeCalib)
+            # Pass it gains of 1 to always stay in the same units.
+            self.deferredChargeCorrection.run(
+                ccdExposure,
+                deferredChargeCalib,
+                gains={key: 1.0 for key in gains},
+            )
```

Comment:
To keep the same behavior as before, shouldn't the gains be 1.0 if electrons and gain values if ADU?  Or do we need to make a change to the `self.deferredChargeCorrection.run()` method to allow the caller to override useGains explicitly?  (Though that is also ugly).